Sources/Cocoa/Components/LabelTextView.swift

```diff
@@ -141,3 +142,32 @@ extension LabelTextView {
         set { super.linkTextAttributes = newValue }
     }
 }
+
+// MARK: - Accessibility
+
+extension LabelTextView {
+    /// The accessibility trait value that the element takes based on the
+    /// text style it was given when created.
+     open override var accessibilityTraits: UIAccessibilityTraits {
+        get {
+            guard
+                let currentFont = font,
+                let textStyle = currentFont.fontDescriptor.object(forKey: .textStyle)
```

Comment:
Hey @kameroli, nice work.

are we able to do this? ⬇️ 

```suggestion
                let textStyle = currentFont.fontDescriptor.object(forKey: .textStyle) as? UIFont.TextStyle
```

Reply:
@mstana thanks for the suggestions, definitely much better way to do it.